### PR TITLE
Dynamic project section

### PR DESCRIPTION
**Pull request by Yosefgeda**
### Javasript-Html

- Changed the HTML of the work section part of the portfolio to JavaScript using DOM.
- Added Modal widow popup which contains projects done.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -13,4 +12,210 @@ closeModalIcon.addEventListener('click', toggleMenuDisplay);
...
+});
+
+const popClose = document.querySelector('.pop-close');
+popClose.addEventListener('click', () => {
+  const popContBig = popContainerBig.style;
+  popContBig.display = 'none';
+  if (popContBig.display === 'none') {
+    bodyMain.style.overflow = 'auto';
+  }
+});
```
Comment (prolajumokeoni) [inline script.js]: - [ ] Kindly make sure that the modal popup works for each project card currently, only the first project modal card works properly, please make sure the other project  cards have modals popup


https://user-images.githubusercontent.com/69638013/216056742-61b6cf91-c609-48cd-926d-0f0da1a26dd1.mp4


- You can  create an array of objects to store JavaScript objects containing the information for each project card, and your modal popup data is populated with the data from the projects array

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @Team
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used 
✔️  Good PR title and Description



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- Kindly ensure that as partners you always have balanced commits in a pair programming project 👍 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (Abenezer-Tilahun) [reply script.js]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @Yosefgeda 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git Hub-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented Suggested changes.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**